### ISSUE TITLE
MutSig and Gistic results

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#4397]
MutSig and Gistic results are displayed in query page and results page. They are currently only available for TCGA provisional (and a few published) studies.

@schultzn @tmazor @ecerami How useful do you think the MutSig and Gistic results are? Do you use them for your analysis?

**Comment by schultzn** [comment]
I think the concept itself is very important - highlighting the genes that are mutated above an expected background. Maybe we should discuss how to do that better going forward?

**Comment by tmazor** [comment]
I agree - I think it's useful for interpreting the data. It would be great if we could somehow extend this to additional studies.

**Comment by schultzn** [comment]
I had a conversation with Marcin Imielinski from Cornell / NYGC. They have a mutational significance method that he thinks can be run on the fly on any data set. Maybe we can consider that as an option. He said he would contact me once it is ready.

In the meantime, I wonder if we could allow users to filter by cancer genes? I assume filtering by only known or likely oncogenic mutations would take too long?

**Comment by imielinski** [comment]
let's do it!  how do we get started?

**Comment by jjgao** [comment]
@imielinski thanks for helping us on this. Would you please give us some pointers of your method, code, or paper? If you have time to help integrate it into the portal code, that'd be even better :)

**Comment by imielinski** [comment]
It’s

https://github.com/mskilab/fishHook

tutorial on lung adenocarcinoma exomes is here

http://mskilab.com/fishHook/tutorial.html

Beyond that - I think we should definitely meet to discuss details!

Marcin

**Comment by jjgao** [comment]
@imielinski thank you for the links. We should meet and discuss. Maybe after summer?

**Comment by imielinski** [comment]
After summer sounds perfect

Best- Marcin
